packages/components/Shimmer/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@fluentui-react-native/shimmer",
+  "version": "0.6.31",
```

Comment:
0.0.1?

Reply:
Right as I was reading this comment, I happened to have this stackoverflow open. It seems 0.1.0 might be the better start 

https://stackoverflow.com/questions/52236193/what-is-the-proper-first-version-0-1-0-or-0-0-1-or-anything-else